tests/standalone_tests/test_parser.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+import tube.settings as config
+
+from tube.utils.dd import init_dictionary
+from unittest.mock import patch
+from tests.util import mock_dictionary_url, initialize_mappings
+from tube.etl.indexers.injection.parser import Parser as InjectionParser
+
+@pytest.fixture(scope="function")
+@patch("tube.etl.indexers.injection.parser.Parser.get_edges_having_data")
+@patch(
+    "dictionaryutils.load_schemas_from_url",
+)
+def test_create_prop_from_json():
+    """
+    Unit test for parser to check if the property created is correct.
+    """
+    from tube.etl.indexers.base.parser import Parser
+
+    dictionary_url_patcher = patch(
+        "dictionaryutils.load_schemas_from_url",
+        return_value=mock_dictionary_url("midrc"),
+    )
+    dictionary_url_patcher.start()
+    dictionary, model = init_dictionary("nor_used_url")
+
+    for e in model.Edge.get_subclasses():
+        print(e.__tablename__)
+
+    mapping = initialize_mappings("midrc", "data_file")
+    parser = InjectionParser(mapping, model, dictionary)
+    for prop in parser.props:
+        assert prop.type is not None
```

Comment:
i think it would be a better test to check the actual values, instead of just checking that they're not null

instead of `for prop in parser.props`  i would pick a few properties to cover all use cases, and check that `prop.type` is the expected type

Reply:
it is unnecessary to check actual values since we have another test to test the parser in dataframe. This test is check to ensure no None type exist